src/app/modal/modal.component.ts

```diff
@@ -7,11 +7,11 @@ import { SchedulesService, UserService, AlertService } from '../services/index';
   templateUrl: './modal.component.html'
 })
 export class ModalComponent implements OnInit {
-  @Input() childData: { id: Number, name: any, key: any, message: any };
+  @Input() childData: { id: Number, name: any, key: any, message: any, protocol: any };
```

Comment:
id > port

Reply:
It can be; but here, `childData` is common for every modal and almost modals use it as id. Only 'Service Health' page use it as port.